client/src/entrypoints/admin/bulk-actions.js

```diff
@@ -101,6 +135,7 @@ function onSelectIndividualCheckbox(e) {
       .querySelectorAll(BULK_ACTION_PAGE_CHECKBOX_INPUT)
       .forEach((el) => el.classList.remove('show'));
   } else if (numCheckedObjects === 1 && prevLength === 0) {
+    // checkedState.prevCheckedObject = null
```

Comment:
```suggestion
```

Just need to remove the commented out line.

Reply:
Done 👍️